.github/workflows/fleet-and-orbit.yml

```diff
@@ -383,6 +383,38 @@ jobs:
       run: |
         "C:\Program Files\Orbit\bin\orbit\orbit.exe" shell -- --json "select * from osquery_info;" | jq -e "if (.[0]) then true else false end"
 
+    - name: Fleet Service Tests
+      shell: powershell
+      run: |
+          #Tests setup
+          $serviceName = "Fleet osquery"
+          $defaultWaitTime = 2
+
+          #Test 1 - Check that the service starts without issues
+          Stop-Service -Name $serviceName
+          Start-Service -Name $serviceName
+          Get-Service -Name $serviceName | %{ if ($_.Status -ne "Running") { throw "Test #1 failed" } }
+
+          #Test 2 - Check that the service stops without issues
+          Stop-Service -Name $serviceName
+          Get-Service -Name $serviceName | %{ if ($_.Status -ne "Stopped") { throw "Test #2 failed" } }
+
+          #Test 3 - Check that no orbit.exe is running after service stop
+          Start-Service -Name $serviceName
+          Stop-Service -Name $serviceName
+          Start-Sleep -Seconds $defaultWaitTime # shutdown wait time
+          Get-Process | %{ if ($_.Name -eq "orbit") { throw "Test #3 failed" } }
+
+          #Test 4 - Check that service starts in less than 3 secs
```

Comment:
Nit: 2 secs?

Reply:
good comment :D. I arbitrarily chose to test that the service was shutting down/starting up in less than 3 seconds, so I'm using a 2-second delay to test the status of the service before those 3 seconds.